org.tenacityaudio.Tenacity.yaml

```
@@ -0,0 +1,176 @@
+app-id: org.tenacityaudio.Tenacity
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: tenacity
+rename-icon: tenacity
+rename-desktop-file: tenacity.desktop
+rename-appdata: tenacity.metainfo.xml
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
```

```suggestion
  - --socket=x11
```

Audio plugin UI will never work otherwise.